### PR TITLE
Set cmake_policy(SET CMP0074 NEW)

### PR DESCRIPTION
**Pull request by janciesko**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@ cmake_minimum_required (VERSION 3.12)
 project(KokkosRemote
   LANGUAGES CXX
   VERSION 0.1.0)
+  
+message(STATUS "Setting policy CMP0074 to use <Package>_ROOT variables")
+cmake_policy(SET CMP0074 NEW)
```
Comment (dalg24) [inline CMakeLists.txt:8]: Why would that be needed?

Comment (dalg24) [reply CMakeLists.txt:8]: your minimum cmake requirement for your project is 3.12 so you don't need it

Comment (janciesko) [reply CMakeLists.txt:8]: Thanks

**Comment by janciesko** [comment]
Closing as obsolete.

**Pull request closed without merging**